hazelcast/security.py

```diff
@@ -1,13 +1,17 @@
 from hazelcast.six import string_types
+from hazelcast.core import Address
 
 
 class TokenProvider(object):
     """TokenProvider is a base class for token providers."""
 
-    def token(self):
-        # type: (TokenProvider) -> bytes
+    def token(self, address):
```

Comment:
Does it make sense to make this optional by providing `None` as the default value? In basic use cases, you don't need an address, you can simply ignore it.

Reply:
Yeah, that'd be better: https://github.com/hazelcast/hazelcast-python-client/pull/455/commits/57f3d488c0cc7b3959134576e29c97f6eddde35d